doc_src/cmds/bind.rst

```diff
@@ -265,6 +265,12 @@ The following special input functions are available:
 ``history-token-search-forward``
     search the history for the next matching argument
 
+``history-last-token-search-backward``
+    search the history for the last argument of the previous command
+
+``history-last-token-search-forward``
+    search the history for the last argument of the next command
```

Comment:
I guess this could be phrased as

	search the history for the next matching last argument